frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -8,10 +8,10 @@ import {
 } from '@console/internal/models';
 import { k8sCreate, K8sResourceKind } from '@console/internal/module/k8s';
 import { SelectorInput } from '@console/internal/components/utils';
+import { createKnativeService } from '@console/knative-plugin/src/utils/create-knative-utils';
```

Comment:
Why move this? It just adds one more thing to be reviewed.

Reply:
had discussed with @christianvogt and IMO it would make more sense to add knative specific utils in knative-plugin as we have now

Reply:
ok